cmd/dp-import-tracker/main.go

```diff
@@ -117,24 +119,29 @@ func CheckImportJobCompletionState(api *api.ImportAPI, jobID, completedInstanceI
 		}
 	}
 	// assert: all instances for jobID are marked "completed"/"error", so update import as same
-	if err := api.UpdateImportJobState(jobID, targetState); err != nil {
+	if err := importAPI.UpdateImportJobState(jobID, targetState); err != nil {
 		log.ErrorC("CheckImportJobCompletionState update", err, log.Data{"jobID": jobID, "last completed instanceID": completedInstanceID})
 	}
 	return nil
 }
 
 // updateInstanceWithObservationsInserted updates a specific import instance with the counts of inserted observations
-func updateInstanceWithObservationsInserted(api *api.ImportAPI, instanceID string, observationsInserted int32) error {
-	if err := api.UpdateInstanceWithNewInserts(instanceID, observationsInserted); err != nil {
+func updateInstanceWithObservationsInserted(datasetAPI *api.DatasetAPI, instanceID string, observationsInserted int32) error {
+	if err := datasetAPI.UpdateInstanceWithNewInserts(instanceID, observationsInserted); err != nil {
```

Comment:
dont need the if, can just `return datasetAPI.Update...` (though at that point this whole function is superfluous)

Reply:
Good point. I have inlined the function 👍